geoportailv3/static/js/draw/featurepopupservice.js

```diff
@@ -159,6 +163,8 @@ app.FeaturePopup.prototype.show = function(feature, opt_anchor) {
   goog.array.forEach(headers, function(element) {
     this.setDraggable(angular.element(element));
   }, this);
+  // Enable the dropdown menu
+  $('.dropdown-toggle')['dropdown']();
```

Comment:
In the following example, I demonstrate that there's no need to call 'dropdown' on elements added via jQuery and an ol.Overlay.
http://jsfiddle.net/pgiraud/36xvpng4/1/

I need to test what happens when the element is created with angular though.